src/Voron/StorageEnvironment.cs

```diff
@@ -391,7 +392,7 @@ internal void TransactionAfterCommit(LowLevelTransaction tx)
             }
 
             Interlocked.Add(ref _sizeOfUnflushedTransactionsInJournalFile, totalPages);
-            if (tx.IsLazyTransaction)
+            if (tx.IsLazyTransaction == false)
```

Comment:
Yes, after a non lazy transaction, we flush to the journal, and then let the flusher know that it is active